### PR TITLE
Feature/fix specific styles

### PR DESCRIPTION
**Pull request by khashayar-pakkhesal**
Fixed specific selectors in css

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTests.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public async Task BitCompoundButtonShouldRespectIsEnabled(bool isEnabled, int co
                     parameters.Add(p => p.BitCompoundButtonStyle, style);
                 });
 
-            var bitButton = com.Find(".bit-compound-button");
+            var bitButton = com.Find(".bit-cmpnd-btn");
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTests.cs:43]: "cmp" is a better abbriviation:  bit-cmp-btn

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI.Tests/SearchBoxes/BitSearchBoxTests.cs [hunk 2]
```diff
@@ -37,7 +37,7 @@ public void SearchBox_NoAnimation_ShouldHaveClassName(bool disableAnimation)
         {
             var component = RenderComponent<BitSearchBoxTest>(parameter =>
             parameter.Add(p => p.DisableAnimation, disableAnimation));
-            var searchBox = component.Find(".bit-search-box-container");
+            var searchBox = component.Find(".bit-srch-box-container");
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI.Tests/SearchBoxes/BitSearchBoxTests.cs:40]: "sch" or any other 3-letters abbreviation

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitButton.razor.cs [hunk 3]
```diff
@@ -23,13 +23,13 @@ public ButtonStyle ButtonStyle
...
-        protected override string RootElementClass => "bit-button";
+        protected override string RootElementClass => "bit-btn";
 
         protected override void RegisterComponentClasses()
         {
             ClassBuilder.Register(() => IsEnabled is false ? string.Empty :
-                                        ButtonStyle == ButtonStyle.Primary ? "primary" :
-                                        "standard");
+                ButtonStyle == ButtonStyle.Primary ? $"{RootElementClass}-primary-{VisualClassRegistrar()}"
+                : $"{RootElementClass}-standard-{VisualClassRegistrar()}");
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitButton.razor.cs:32]: improve this ternary operator code formatting

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Checkboxes/BitCheckbox.razor [hunk 4]
```diff
@@ -10,15 +10,15 @@
            disabled="@(IsEnabled is false)"
            hidden>
 
-    <div class="bit-checkbox-item bit-checkbox-item__checkbox" @onclick="@HandleCheckboxClick">
-        <i class="bit-checkbox-item__checkmark"></i>
-    </div>
+    <div class="bit-chb-check-wrapper-fluent">
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Checkboxes/BitCheckbox.razor:13]: i think these CSS classes are redundant

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor.cs [hunk 5]
```diff
@@ -74,21 +74,21 @@ public string FocusClass
             }
         }
 
-        protected override string RootElementClass => "bit-text-field";
+        protected override string RootElementClass => "bit-txt-fld";
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor.cs:77]: I think "bit-txt" is fine

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Links/BitLink.razor.cs [hunk 6]
```diff
@@ -14,7 +15,7 @@ public partial class BitLink
 
         [Parameter] public EventCallback<MouseEventArgs> OnClick { get; set; }
 
-        protected override string RootElementClass => "bit-link";
+        protected override string RootElementClass => Href.HasValue() || !IsEnabled ? "bit-lnk"  : "bit-lnk-btn";
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Links/BitLink.razor.cs:18]: what is this change for?

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/SearchBoxes/BitSearchBox.razor [hunk 7]
```diff
@@ -4,7 +4,7 @@
 <div @ref="RootElement"
      style="@StyleBuilder.Value"
      class="@ClassBuilder.Value">
-    <div class="search-icon-container">
+    <div>
         <i class="bit-icon bit-icon--@IconName"></i>
     </div>
     <input type="text" @ref="InputRef" class="search-input"
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/SearchBoxes/BitSearchBox.razor:10]: redundant CSS classes

File: src/Client/Web/Bit.Client.Web.BlazorUI.Tests/SearchBoxes/BitSearchBoxTests.cs (hunk 2 above)
Comment (khashayar-pakkhesal) [reply src/Client/Web/Bit.Client.Web.BlazorUI.Tests/SearchBoxes/BitSearchBoxTests.cs]: bit-search-box-container changed to bit-sch-box-container

File: src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTests.cs (hunk 1 above)
Comment (khashayar-pakkhesal) [reply src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTests.cs]: bit-compound-button changed to bit-cmp-btn

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitButton.razor.cs (hunk 3 above)
Comment (khashayar-pakkhesal) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitButton.razor.cs]: Fixed all ternary  operators formatting

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Checkboxes/BitCheckbox.razor (hunk 4 above)
Comment (khashayar-pakkhesal) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Checkboxes/BitCheckbox.razor]: bit-chb-check-wrapper-fluent was removed and all styles are added directly from parent

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor.cs (hunk 5 above)
Comment (khashayar-pakkhesal) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor.cs]: fld is removed from class name

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Links/BitLink.razor.cs (hunk 6 above)
Comment (khashayar-pakkhesal) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Links/BitLink.razor.cs:18]: an issue is made for this change 
https://github.com/bitfoundation/bitframework/issues/310

**Review by msynk — APPROVED** [review]

**Pull request merged**